packages/apify/src/storages/dataset.ts

```diff
@@ -227,9 +225,10 @@ export class Dataset {
      *   If `true` then the function doesn't return empty items.
      *   Note that in this case the returned number of items might be lower than limit parameter and pagination must be done using the `limit` value.
      */
-    async getData<T = unknown>(options = {}): Promise<DatasetContent<T>> {
+    async getData(options = {}): Promise<DatasetContent<Data>> {
```

Comment:
let's add a TODO here so we know we need to create the options interface, and for now at least use `Dictionary`